src/components/Categories.js

```diff
@@ -1,5 +1,18 @@
 import React from 'react';
+import { useDispatch } from 'react-redux';
+import { checkStatus } from '../redux/categories/categories';
 
-const Categories = () => (<button className="btn-categories" type="button">Check status</button>);
+const Categories = () => {
+  const dispatch = useDispatch();
+    <button
+      className="btn-categories"
+      onClick={() => {
+        dispatch(checkStatus());
+      }}
+      type="button"
+    >
+      Check status
+    </button>;
+};
```

Comment:
- [ ] The button that checks the status is not rendered by the Categories component. I would recommend adding `return` to the Categories component so the button appears on the categories page.